ui/main.sh

```diff
@@ -273,7 +273,6 @@ if [[ "${get[page]}" == "main" && "${get[section]}" == "start" ]]; then
 								IFS="
 								"
 								for backupconfig in ${backupconfigs}; do
-									IFS="$backifs"
```

Comment:
This line becomes obsolete because the variable backifs is not assigned. In line 273 IFS is assigned with line feed and it is reest at line 276 with an empty string.
Therefore my opionion is that the line can be removed.

Reply:
Create new commit with correct assignment to IFS.